lib/posix-user/musl-imported/include/grp.h

```diff
@@ -15,33 +15,171 @@ extern "C" {
 
 #include <sys/types.h>
 
+/**
+ * An entry containing the necessary data for defining a group
+ */
 struct group {
+	/* Group name */
 	char *gr_name;
+	/* Group password */
 	char *gr_passwd;
+	/* Group identifier */
 	gid_t gr_gid;
+	/* List of group members */
 	char **gr_mem;
 };
 
+/**
+ * Searches through the groups (there is just the default group at the moment)
+ * for a specific group ID and returns a pointer to an object with the group
+ * information.
+ * This is part of the POSIX documentation, refer to that for more details.
+ *
+ * @param gid
+ *   Group id to search by
+ * @return
+ *   Pointer to the group with group id `gid`.
+ */
 struct group  *getgrgid(gid_t);
+
+/**
+ * Searches through the groups (there is just the default group at the moment)
+ * for a specific group name and returns a pointer to an object with the group
+ * information.
+ * This is part of the POSIX documentation, refer to that for more details.
+ *
+ * @param name
+ *   Group name to search by
+ * @return
+ *   Pointer to the group with group name `name`.
+ */
 struct group  *getgrnam(const char *);
 
+/**
+ * Like `getgrgid`, but stores the retrieved `group`
+ * structure in the space pointed to by `grp`.
+ * This is part of the POSIX documentation, refer to that for more details.
+ *
+ * @param gid
+ *   Group id to search by
+ * @param grp
+ *	 Group structure in which the group with the `gid` group id is stored
```

Comment:
```suggestion
 *   Group structure in which the group with the `gid` group id is stored
```
Do not use tabs within comment blocks.